tests/quota/awsNodeSdk.js

```diff
@@ -338,11 +338,18 @@ function multiObjectDelete(bucket, keys, size, callback) {
         };
         mockScuba = scuba;
 
-        before(done => {
+        before(() => {
             const config = getConfig('default', { signatureVersion: 'v4', maxRetries: 0 });
             s3Client = new S3(config);
             scuba.start();
-            return metadata.setup(err => wait(2000, () => done(err)));
+            return new Promise((resolve, reject) => {
+                metadata.setup(err => {
+                    if (err) {
+                        reject(err);
+                    }
+                    wait(2000, resolve);
+                });
+            });
```

Comment:
do we really _need_ to change the callback-based hook here?
as we are still using only callback-based functions internally, the updated code is more complex than the old one: if no constraint, may be better to keep it until those functions have been updated